app/src/main/java/com/quarkworks/apartmentgroceries/MainActivity.java

```diff
@@ -1,55 +1,63 @@
 package com.quarkworks.apartmentgroceries;
 
 import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.AsyncTask;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.view.Menu;
-import android.view.MenuItem;
 
 import com.quarkworks.apartmentgroceries.auth.LoginActivity;
-import com.quarkworks.apartmentgroceries.auth.SignupActivity;
-import com.quarkworks.apartmentgroceries.user.UserActivity;
+import com.quarkworks.apartmentgroceries.main.HomeActivity;
 
 
 public class MainActivity extends AppCompatActivity {
 
-    private static final String LOG_TAG = MainActivity.class.getSimpleName();
+    private static final String TAG = MainActivity.class.getSimpleName();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main_activity);
+        new BackgroundSplashTask().execute();
     }
 
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
-        getMenuInflater().inflate(R.menu.main_menu, menu);
-        return true;
-    }
-
-    @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
+    private class BackgroundSplashTask extends AsyncTask<Void, Void, Boolean> {
```

Comment:
Probably don't need an AsyncTask for this. It is pretty quick, so you can do it in onCreate().